source/developers/experience-builder.rst

```diff
@@ -1139,65 +1208,468 @@ breadcrumb
+       disable XB options that aren't relevant to headless application such as SPAs.
+   * - ``themeOptions``
+     - `MUI's ThemeOptions <https://mui.com/customization/theming>`__
+     - XB's defaults
+     - XB is powered by MUI. This argument allows you to customize MUI theme options and override
+       XB's defaults.
+   * - ``sxOverrides``
+     - ExperienceBuilderStylesSx
+     - XB's defaults
+     - You may change XB-specific theming through this argument
+   * - ``documentDomain``
+     - string
+     - null
+     - You may specify a ``documentDomain`` if your preview runs on a different domain than Studio does.
+   * - ``scrollElement``
+     - string
+     - html, body
+     - You may specify a different element for XB to scroll when scrolling the user to specific
+       CrafterCMS field elements.
+
+Model
+"""""
+
+Use this component to render elements that represent the **models themselves** (i.e. CrafterCMS pages or
+components, not their fields).
+
+.. list-table::
+   :widths: 10 10 10 70
+   :header-rows: 1
+
+   * - Prop
+     - Type
+     - Default
+     - Description
+   * - ``model``
+     - Object (ContentInstance)
+     - (Required)
+     - The model being rendered
+   * - ``component``
+     - string | React.ElementType
+     - "div"
+     - The component to be rendered
+   * - ``componentProps``
+     - Object
+     - undefined
+     - Any props sent at the root that aren't own props are forwarded down to the rendered
+       component so in most cases you needn't use ``componentProps``. There may be cases where your
+       target component has a prop name that matches in name with a prop of the CrafterCMS React
+       component so to avoid it swallowing the prop and not reaching your target component, you may
+       send the prop(s) via ``componentProps`` instead.
+
+ContentType
+"""""""""""
+
+Use this component to render a specific component of your own library based on the content type of the
+model. ``ContentType`` component works with a "content type map" which you must supply as a prop. The
+content type map, is essentially a plain object, a lookup table of your components indexed by content
+type id. You may use it in conjunction with ``React.lazy`` to optimize your app; specially considering the
+content type map should contain all the possible components that you will be rendering via ``ContentType``
+component on a given piece of your app.
+
+.. list-table::
+   :widths: 10 10 10 70
+   :header-rows: 1
+
+   * - Prop
+     - Type
+     - Default
+     - Description
+   * - ``model``
+     - Object (ContentInstance)
+     - (Required)
+     - The model being rendered
+   * - ``contentTypeMap``
+     - Object
+     - (Required)
+     - A map of components indexed by CrafterCMS content type id. The content type id of the model passed
+       will is used to pick from the map the component that should render said model.
```

Comment:
Should it say `The content type id of the model passed will be used to pick...` ?